### PR TITLE
habitus 1.0.1 (new formula)

### PR DESCRIPTION
**Pull request by itsmechlark**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/habitus.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+class Habitus < Formula
+  desc "A Build Flow Tool for Docker"
+  homepage "http://www.habitus.io"
+  url "https://github.com/cloud66/habitus/releases/download/1.0.1/habitus_darwin_amd64"
+  version "1.0.1"
```
Comment (cglong) [inline Formula/habitus.rb]: Do we need to explicitly define the `version`? It should normally be inferable by the `url`.

File: Formula/habitus.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
+class Habitus < Formula
+  desc "A Build Flow Tool for Docker"
+  homepage "http://www.habitus.io"
+  url "https://github.com/cloud66/habitus/releases/download/1.0.1/habitus_darwin_amd64"
```
Comment (cglong) [inline Formula/habitus.rb]: This needs to be built from source.

File: Formula/habitus.rb (hunk 1 above)
Comment (ilovezfs) [reply Formula/habitus.rb]: It's confused.
```
Josephs-MacBook-Pro:~ joe$ brew irb
==> Interactive Homebrew Shell
Example commands available with: brew irb --examples
irb(main):001:0> Version.detect("https://github.com/cloud66/habitus/releases/download/1.0.1/habitus_darwin_amd64", {})
=> #<Version::FromURL:0x007fd85284b980 @version="64">
```
@cglong note `brew audit` will puke if an explicit version is redundant.

**Comment by ilovezfs** [comment]
@itsmechlark please feel free to open a new PR that builds this from source.

**Pull request closed without merging**